lib/kitchen/transport/speedy_ssh.rb

```diff
@@ -28,12 +28,12 @@ class Connection < Ssh::Connection
         include SpeedyConnectionBase
 
         def valid_local_requirements?
-          !Mixlib::ShellOut.new("which tar > /dev/null").run_command.error?
+          !Mixlib::ShellOut.new("which tar && which gzip > /dev/null").run_command.error?
```

Comment:
which tar > /dev/null && which gzip > /dev/null

(else only taken into account for last command, not sure there is another way to make it happen both in a single > )